GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -458,6 +465,8 @@ public async Task Onboard([QueueTrigger("onboarding")] string queueItem, Executi
         [FunctionName("TrafficTimer")]
         public Task TrafficTimer([TimerTrigger("0 0 8 * * *" /* run once every day at 00:00:00 PST*/)] TimerInfo timerInfo, ExecutionContext executionContext, ILogger logger)
         {
+            using Activity invocationActivity = GetInvocationActivity(executionContext, timerInfo).Start();
```

Comment:
Same here (and below) as in the ADO collector. The Activity is going to be disposed immediately because we are not `await`ing on anything.